Add roadworks list slice

chrisrhymes opened the pull request:
Resolves https://github.com/FutureNorthants/northants-website/issues/1213

## Changes
- Add roadworks list slice
- Update storybook version

## Testing
- Checkout this branch with `git fetch && git checkout 1213-display-a-table-of-roadworks`
- Run `npm install` to update dependencies, then run `npm run dev`
- Visit the Slices ->Roadworks List` 
- Test the search filters the list
- Run tests with `npm run test`

mstanford-wnc commented:
Hi @chrisrhymes  I've added additional Attributes to the <Input> search-box to address the three silktide accessibility issues highlighted in the screenshot below, and have committed them accordingly.  Also added to <Input> were the 'type' attribute (value 'text') and a 'maxLength' value of 255.

![image](https://github.com/user-attachments/assets/b826e3b7-ce2c-4b3c-a53a-ff1a89a72fbe)

chrisrhymes commented:
> Hi @chrisrhymes I've added additional Attributes to the search-box to address the three silktide accessibility issues highlighted in the screenshot below, and have committed them accordingly. Also added to were the 'type' attribute (value 'text') and a 'maxLength' value of 255.

Great work @mstanford-wnc! Well spotted!
